shm/Map.hpp

```diff
@@ -3,11 +3,17 @@
 #include "Island.hpp"
 #include <vector>
 
+constexpr size_t amountIslands = 10;
+constexpr size_t heightIsland = 10;
+constexpr size_t widthIsland = 10;
```

Comment:
As I wrote earlier, I don't quite get the idea behind height and width variables.

Another thing is that I'm not sure if for example `amountIsland` should be here or maybe it should be a private member of Map class? But as I said - I don't really know for sure.

Reply:
Name and values of these consts should change. They should represent size of a map. 
`mapHeight`, `mapWidth`. So for example Map of a size 100x100  means `mapHeight=100`, `mapWidth=100`.

Reply:
And I think for now they can stay as constexpr outside of class. Although the naming is a little awkward. `amountOfIslands` looks better.